org.metaborg.parsetable/src/main/java/org/metaborg/parsetable/characterclasses/CharacterClassSingle.java

```diff
@@ -4,6 +4,7 @@
 
 import org.spoofax.interpreter.terms.IStrategoTerm;
 import org.spoofax.interpreter.terms.ITermFactory;
+import org.spoofax.terms.util.B;
```

Comment:
Can you also remove the unused imports? :innocent:

Reply:
Done